### DISCUSSION TITLE
How to use `ssh` for cloning a plugin repository

### DISCUSSION THREAD

**Discussion by ghost** [zdharma-continuum/zinit#522] [Q&A] [answered]
### If the feature solves a problem you have, specify it here.

Since GitHub changes the authentication from password to ssh, I switch the clone method from https to ssh, when using https to download plugins is so slow and have ssl error sometimes

### Describe the proposed feature.

Maybe can add option for change the default download method, can switch to https or ssh

### Related Issues

#59

**Comment by vladdoster** [comment, marked as answer]
I think this is already possible via the `proto` ice?

https://github.com/zdharma-continuum/zinit#cloning-options

Also, `https` and `ssh` have no real speed difference since TCP/UDP IP is under the hood. Additionally, `https` can be much faster if in a high-latency environment. `https` has been optimized much more than SSH.

**Comment by ghost** [comment]
I just read the manual and search others snippet, I configure ssh clone worked, thanks the answer @vladdoster
